Protocol.md

```diff
@@ -279,6 +281,58 @@ The recipient:
 
   These script pub key forms include only standard forms accepted by the wider set of deployed Bitcoin clients in the network, which increase the chances of successful propagation to miners.
 
+## Optional features
+
+Optional features consist of "features" which parties can opt-in to using. These messages are not needed to facilitate construction of a DLCs.
+
+### The `close_dlc` Message
+
+This message contains information about mutual close, which allows the counterparty
+to broadcast a mutual closing transaction.
+
+1. type: ? (`close_dlc_v0`)
+2. data:
+   * [`32*byte`:`contract_id`]
+   * [`signature`:`close_signature`]
+   * [`u64`:`offerPayoutSatoshis`]
+   * [`u64`:`acceptPayoutSatoshis`]
+   * [`u64`:`fund_input_serial_id`]
+   * [`u16`:`num_extra_inputs`]
+   * [`num_extra_inputs*extra_input`:`extra_inputs`]
+   * [`extra_signatures`:`extra_signatures`]
```

Comment:
This is not a defined type. I guess you wanted to do the same as for `funding_signatures`, but note that `funding_signatures` is actually defined in `Messaging.md`. Maybe the best would be to change the type name of `funding_signatures` so that it's more abstract and reuse that (if the goal is to have the exact same structure). I guess originally there was only a single use for it which is why it was named like that.

Reply:
Renamed `funding_input` to `input` and `funding_signatures` to `input_signatures` to abstract them from funding transaction and allow them to be re-used for `DlcClose` messages.

Reply:
Thanks! I might have gone with `tx_input` instead of just `input` but we can change it later if the meaning is not clear enough.